pingdom-checks/address-validation.ping

```diff
@@ -1,26 +1,32 @@
 # -*- sh -*-
 
+SANDBOX_ADDRESS_VALIDATION_API_KEY=$(get-secret "/dvp/lab/address-validation/api-key")
```

Comment:
this is nitpicky and maybe a team discussion, but I feel like the param store path should be `/dvp/address-validation/lab/api-key`? That way all of the values for `address-validation` are in one place and it's easy enough to go into the individual environments from there. I know the current naming convention seems to be what you have here, but figured I'd bring it up - @mramirez-va

Reply:
You have a point about grouping by API. Although it's mostly robots looking at these params so I'm cool with the current convention.